### PR TITLE
Add AllBe for string collections

### PR DESCRIPTION
**Pull request by rynkevich**
Implements feature #1263.

## IMPORTANT 

* [x] The code complies with the [Coding Guidelines for C#](https://www.csharpcodingguidelines.com/).
* [x] The changes are covered by a new or existing set of unit tests which follow the Arrange-Act-Assert syntax such as is used [in this example](https://github.com/fluentassertions/fluentassertions/blob/daaf35b9b59b622c96d0c034e8972a020b2bee55/Tests/FluentAssertions.Shared.Specs/BasicEquivalencySpecs.cs#L33).
* [x] If the contribution affects [the documentation](https://github.com/fluentassertions/fluentassertions/tree/master/docs/_pages), please include your changes in this pull request so the documentation will appear on the [website](https://www.fluentassertions.com).

### CODE REVIEW

File: Src/FluentAssertions/Collections/StringCollectionAssertions.cs [hunk 1]
```diff
@@ -158,18 +155,15 @@ public AndConstraint<TAssertions> BeEquivalentTo(IEnumerable<string> expectation
...
+        public AndConstraint<TAssertions> AllBe(string expectation,
             string because = "", params object[] becauseArgs)
         {
-            return AllBeEquivalentTo(expectation, options => options, because, becauseArgs);
+            return AllBe(expectation, options => options, because, becauseArgs);
         }
 
         /// <summary>
-        /// Asserts that all elements in a collection of objects are equivalent to a given object.
+        /// Asserts that all elements in a collection of objects are equal to a given object.
```
Comment (dennisdoomen) [inline Src/FluentAssertions/Collections/StringCollectionAssertions.cs:165]:
```suggestion
        /// Asserts that all strings in a collection of strings are equal to the given string.
```

File: Tests/FluentAssertions.Specs/Equivalency/CollectionEquivalencySpecs.cs [hunk 2]
```diff
@@ -828,197 +828,197 @@ public void When_a_strongly_typed_collection_is_declared_as_an_untyped_collectio
         }
 
         [Fact]
-        public void When_all_subject_items_are_equivalent_to_expectation_object_it_should_succeed()
+        public void When_all_string_subject_items_are_equal_to_expectation_object_it_should_succeed()
```
Comment (dennisdoomen) [inline Tests/FluentAssertions.Specs/Equivalency/CollectionEquivalencySpecs.cs:831]:
```suggestion
        public void When_all_strings_in_the_collection_are_equal_to_the_expected_string_it_should_succeed()
```

**Comment by rynkevich** [comment]
@dennisdoomen, @jnyrup thanks for the review!
I believe I've made all required adjustments and additions.

**Comment by dennisdoomen** [comment]
Looks like you still need to accept the API changes using `AcceptApiChanges.ps1`

**Pull request merged**